langgraph/pregel/__init__.py

```diff
@@ -760,10 +760,11 @@ def stream(
                             )
                         )
                         checkpoint_config = {
+                            **checkpoint_config,
```

Comment:
Hey @nfcampos, I forgot why we're doing this. Can you explain again? Thank you!

Reply:
This is to ensure all checkpoints after the first contain the same metadata as the first one